src/ripple/ledger/impl/OpenView.cpp

```diff
@@ -45,8 +45,9 @@ class OpenView::txs_iter_impl : public txs_type::iter_base
     bool
     equal(base_type const& impl) const override
     {
-        auto const& other = dynamic_cast<txs_iter_impl const&>(impl);
-        return iter_ == other.iter_;
+        if (auto p = dynamic_cast<txs_iter_impl const*>(&impl))
```

Comment:
Consider `auto const p`.  Not that it makes any difference right now...

Reply:
I fixed this and did the same fix in `Ledger::txs_iter_impl`, which I'd previously missed.